Mage.Tests/src/test/java/org/mage/test/cards/continuous/OpalescenceTest.java

```diff
@@ -0,0 +1,67 @@
+package org.mage.test.cards.continuous;
```

Comment:
Everything else looks good, could you please move this into the [UDS] folder rather than the continuous ability folder.

Reply:
Done